CMakeLists.txt

```diff
@@ -1,18 +1,41 @@
-cmake_minimum_required(VERSION 2.8.3)
+cmake_minimum_required(VERSION 3.5)
 project(rqt_service_caller)
-# Load catkin and all dependencies required for this package
-find_package(catkin REQUIRED)
-catkin_package()
-catkin_python_setup()
+
+find_package(ament_cmake REQUIRED)
+find_package(ament_cmake_python REQUIRED)
+find_package(rqt_py_common REQUIRED)
+find_package(rosidl_default_generators REQUIRED)
+
+set(srv_files
+  "srv/Sum.srv"
+)
```

Comment:
This service files shouldn't be installed.

It should probably be even be removed - see other comment.